docs/sdk/server-side-sdks/node/node-openfeature.md

```diff
@@ -19,18 +19,18 @@ DevCycle provides a NodeJS implementation of the [OpenFeature](https://openfeatu
 
 ### Installation
 
-Install the OpenFeature JS SDK, DevCycle NodeJS Server SDK, and DevCycle NodeJS Provider:
+Install the DevCycle OpenFeature NodeJS Provider:
 
 ```bash
-npm install --save @devcycle/openfeature-nodejs-provider @devcycle/nodejs-server-sdk @openfeature/js-sdk
+npm install --save @devcycle/openfeature-nodejs-provider
 ```
 
 ### Getting Started
 
 Initialize the DevCycle SDK and set the DevCycleProvider as the provider for OpenFeature:
 
 ```typescript
-import { OpenFeature, Client } from '@openfeature/js-sdk'
+import { OpenFeature, Client } from '@openfeature/server-sdk'
```

Comment:
you still need to install the @openfeature/server-sdk package if you're using it in your own application code, otherwise importing it here would only work if you were using `npm` to install packages cuz it hoists things to the root of `node_modules`. This code wouldn't work in `pnpm`

Reply:
ok, I'll revert it to what it was then.

Reply:
you still don't need our node SDK installed manually

Reply:
you do, its being imported in the example there.

Reply:
why? all calls should be going through openfeature

Reply:
Read the example, the node provider was setup as an external package to the node SDK so the SDK is initialized and passed into the provider which is passed into OF.